iothub/service/samples/readme.md

```diff
@@ -1,14 +1,27 @@
+## Service samples for Azure IoT Service SDK for C#
+
 Service samples were moved to [Azure-Samples/azure-iot-samples-csharp][samples-repo] repository
 
 ### [Service samples][service-samples]
-* [ADMSample][adm-sample]
-* [JobsSample][jobs-sample]
-* [RegistryManagerSample][reg-man-sample]
-* [ServiceClientSample][service-client-sample]
+* [ADM Sample][adm-sample]
+* [Device Streaming Sample][device-streaming-sample]
+* [Edge Deployment Sample][edge-deployment-sample]
+* [Import Export Devices Sample][import-export-sample]
+* [Jobs Sample][jobs-sample]
+* [Registry ManagerSample][reg-man-sample]
+* [Service Client Sample][service-client-sample]
+* [Plug and play Samples ][pnp-service-sample]
+* [Digital Twin Client Samples][digital-twin-client-sample]
+
 
 [samples-repo]: https://github.com/Azure-Samples/azure-iot-samples-csharp
 [service-samples]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service
 [adm-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/AutomaticDeviceManagementSample
+[device-streaming-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/DeviceStreamingSample
+[edge-deployment-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/EdgeDeploymentSample
+[import-export-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/ImportExportDevicesSample
 [jobs-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/JobsSample
 [reg-man-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/RegistryManagerSample
 [service-client-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/ServiceClientSample
+[pnp-service-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/PnpServiceSamples
+[digital-twin-client-sample]: https://github.com/Azure-Samples/azure-iot-samples-csharp/tree/master/iot-hub/Samples/service/DigitalTwinClientSamples
```

Comment:
This sample get deprecated soon, right @ericmitt? Should we remove it from this samples readme?